R/driver_rds.R

```diff
@@ -186,6 +187,10 @@ R6_driver_rds <- R6::R6Class(
       }
       self$hash_algorithm <- driver_rds_config(path, "hash_algorithm",
                                                hash_algorithm, "md5", TRUE)
+
+      self$hash_length <- nchar(
```

Comment:
This looks like the sort of thing that will cause me a lot of confusion in future (all of the hash stuff needs an overhaul at some point).  An alternative approach would be an integer vector in `hash.R` that contains the lengths - they won't change!

```r
hash_length = c(md5 = 32L, ...)
```

and just look this up here (`self$hash_length <- hash_length[[self$hash_algorithm]]`)

Or just run `nchar(digest::digest(as.raw(0x00), self$hash_algorithm, serialize = FALSE))`

Reply:
Sure, I like `nchar(digest::digest(...))`. See 21b2966c1e7d8aa055804bb3a6eea382f5c3d4f5.